net/eoip/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2006-2012 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=eoip
+PKG_VERSION:=0.5
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Bogdik <bogdikxxx@gmail.com>
+PKG_LICENSE:=GPL-2.0
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://github.com/bogdik/openwrt-linux-eoip/releases/download/0.6.1/
+PKG_MD5SUM:=ab6afb3eb806aee7e6dbfad2bf74de9b
```

Comment:
From automatic CI testing:
* WARNING: Use of PKG_MD5SUM is deprecated, switch to PKG_HASH
* WARNING: PKG_MD5SUM uses deprecated hash, set to 22f6f3864665adef26c7fbd57543a396108ba2dff1282af8143f18bc2a9912f8

-->
PKG_HASH:=22f6f3864665adef26c7fbd57543a396108ba2dff1282af8143f18bc2a9912f8

Reply:
That's a SHA-256 right?

Reply:
Yep.
PKG_MD5SUM has been deprecated years ago